pkg/notify/notifier/pushover/README.md

```diff
@@ -216,8 +216,6 @@ The message push sends messages in parallel to each user registered by each tena
 
 Preprocess the message before sending it. First, the tenants that subscribed Pushover messages are selected based on the AlertSelector given at the time of resource definition. Second, Pushover has a limit of 1024 characters on the message length (the exceeded part will be truncated), and each message may contain more than one Alert. Thus, a strategy of splitting the message is applied here, i.e., a message should contain as many Alerts as possible, and each message is sent one after another to ensure that they can be received in an intact manner by the user. Third, fit the message to a template, from which the Pushover message structure is constructed and its legitimacy is verified. Finally, the Pushover message structure is encoded as a JSON string, a POST method is called to send a request to the Endpoint (https://api.pushover.net/1/messages.json), and an error will be raised if the status code of the returned response is not successful.
```

Comment:
I think it's better to put these implementation details into corresponding source code as good comments. And the Chinese version is not necessary. The description could be adjusted as below:

```
The preprocess logic of the pushover notification messages are as below:
- The alert messages are filtered by AlertSelector first before sending to Pushover. 
- Pushover has a limit of 1024 characters on the message length (the exceeded part will be truncated), and each message may contain more than one Alert. Thus, a strategy of splitting the message is applied here, i.e., a message should contain as many Alerts as possible, and each message is sent one after another to ensure that they can be received in an intact manner by the user. 
- Render the message with a template, from which the Pushover message structure is constructed and its legitimacy is verified. 
- The Pushover message structure is encoded as a JSON string, a POST method is called to send a request to the Endpoint (https://api.pushover.net/1/messages.json), and an error will be raised if the status code of the returned response is not successful.
```

Reply:
removed README, README_CN & added these comments to src.
https://github.com/txfs19260817/notification-manager/blob/22c8c34141e2ba5bf22d0439cfad8ebdbf9b5ca3/pkg/notify/notifier/pushover/pushover.go#L126